src/zos_console/zowe/zos_console_for_zowe_sdk/response/console.py

```diff
@@ -0,0 +1,49 @@
+"""Zowe Python Client SDK.
+
+This program and the accompanying materials are made available under the terms of the
+Eclipse Public License v2.0 which accompanies this distribution, and is available at
+
+https://www.eclipse.org/legal/epl-v20.html
+
+SPDX-License-Identifier: EPL-2.0
+
+Copyright Contributors to the Zowe Project.
+"""
+from dataclasses import dataclass
+from typing import Any, Optional
+
+
+@dataclass
+class IssueCommandResponse:
+    cmd_response_key: Optional[str] = None
+    cmd_response_url: Optional[str] = None
+    cmd_response_uri: Optional[str] = None
+    cmd_response: Optional[str] = None
+
+    def __init__(self, response: dict) -> None:
+        for k, value in response.items():
+            key = k.replace("-", "_")
+            super().__setattr__(key, value)
+
+    def __getitem__(self, key: str) -> str:
+        return self.__dict__[key.replace("-", "_")]
+
+    def __setitem__(self, key: str, value: str) -> None:
+        self.__dict__[key.replace("-", "_")] = value
```

Comment:
These seem like common functions that are shared across all of the response dataclasses. If so, does it make sense to move them into a base class in the Core SDK that all the other classes can inherit?
```python
class BaseResponse:
    def __init__(self, response: dict) -> None:
        for k, value in response.items():
            key = k.replace("-", "_")
            super().__setattr__(key, value)

    def __getitem__(self, key: str) -> str:
        return self.__dict__[key.replace("-", "_")]

    def __setitem__(self, key: str, value: str) -> None:
        self.__dict__[key.replace("-", "_")] = value
```

Reply:
Only three classes use the exact same common functions. Other classes need to adjust this code based on their attributes. Although I like this idea, I hardly think it would be that helpful to create a separate base class.